app/keepers/keepers.go

```diff
@@ -614,6 +626,118 @@ func NewAppKeepers(
+		"/juno.mint.Query/AnnualProvisions": &minttypes.QueryAnnualProvisionsResponse{},
+
+		// nft
+		"/cosmos.nft.v1beta1.Query/Balance": &nft.QueryBalanceResponse{},
+		"/cosmos.nft.v1beta1.Query/Owner":   &nft.QueryOwnerResponse{},
+		"/cosmos.nft.v1beta1.Query/Supply":  &nft.QuerySupplyResponse{},
+		"/cosmos.nft.v1beta1.Query/NFTs":    &nft.QueryNFTsResponse{},
+		"/cosmos.nft.v1beta1.Query/NFT":     &nft.QueryNFTResponse{},
+		"/cosmos.nft.v1beta1.Query/Class":   &nft.QueryClassResponse{},
+		"/cosmos.nft.v1beta1.Query/Classes": &nft.QueryClassesResponse{},
+
+		// slashing
+		"/cosmos.slashing.v1beta1.Query/Params":       &slashingtypes.QueryParamsResponse{},
+		"/cosmos.slashing.v1beta1.Query/SigningInfo":  &slashingtypes.QuerySigningInfoResponse{},
+		"/cosmos.slashing.v1beta1.Query/SigningInfos": &slashingtypes.QuerySigningInfosResponse{},
+
+		// staking
+		"/cosmos.staking.v1beta1.Query/Validators":                    &stakingtypes.QueryValidatorsResponse{},
+		"/cosmos.staking.v1beta1.Query/Validator":                     &stakingtypes.QueryValidatorResponse{},
+		"/cosmos.staking.v1beta1.Query/ValidatorDelegations":          &stakingtypes.QueryValidatorDelegationsResponse{},
+		"/cosmos.staking.v1beta1.Query/ValidatorUnbondingDelegations": &stakingtypes.QueryValidatorUnbondingDelegationsResponse{},
+		"/cosmos.staking.v1beta1.Query/Delegation":                    &stakingtypes.QueryDelegationResponse{},
+		"/cosmos.staking.v1beta1.Query/UnbondingDelegation":           &stakingtypes.QueryUnbondingDelegationResponse{},
+		"/cosmos.staking.v1beta1.Query/DelegatorDelegations":          &stakingtypes.QueryDelegatorDelegationsResponse{},
+		"/cosmos.staking.v1beta1.Query/DelegatorUnbondingDelegations": &stakingtypes.QueryDelegatorUnbondingDelegationsResponse{},
+		"/cosmos.staking.v1beta1.Query/Redelegations":                 &stakingtypes.QueryRedelegationsResponse{},
+		"/cosmos.staking.v1beta1.Query/DelegatorValidators":           &stakingtypes.QueryDelegatorValidatorsResponse{},
+		"/cosmos.staking.v1beta1.Query/DelegatorValidator":            &stakingtypes.QueryDelegatorValidatorResponse{},
+		"/cosmos.staking.v1beta1.Query/HistoricalInfo":                &stakingtypes.QueryHistoricalInfoResponse{},
+		"/cosmos.staking.v1beta1.Query/Params":                        &stakingtypes.QueryParamsResponse{},
+		"/cosmos.staking.v1beta1.Query/Pool":                          &stakingtypes.QueryPoolResponse{},
+
+		// upgrade
+		"/cosmos.upgrade.v1beta1.Query/CurrentPlan":                 &upgradetypes.QueryCurrentPlanResponse{},
+		"/cosmos.upgrade.v1beta1.Query/AppliedPlan":                 &upgradetypes.QueryAppliedPlanResponse{},
+		"/cosmos.upgrade.v1beta1.Query/QueryModuleVersionsResponse": &upgradetypes.QueryModuleVersionsResponse{},
+		"/cosmos.upgrade.v1beta1.Query/QueryAuthorityResponse":      &upgradetypes.QueryAuthorityResponse{},
+	}
+
+	updated := asq
+	for k, v := range std {
```

Comment:
This is fine since we are iterating all keys & not exiting early (else we would need an ordered map with []string{}'s deterministic iter